fv3core/stencils/d_sw.py

```diff
@@ -93,76 +94,225 @@ def horizontal_relative_vorticity_from_winds(
         vorticity[0, 0, 0] = rarea * (vt - vt[0, 1, 0] - ut + ut[1, 0, 0])
 
 
-@gtstencil()
-def not_inlineq_pressure(gx: sd, gy: sd, rarea: sd, fx: sd, fy: sd, pt: sd, delp: sd):
-    with computation(PARALLEL), interval(...):
-        pt = flux_integral(
-            pt, delp, gx, gy, rarea
-        )  # TODO: Put [0, 0, 0] on left when gt4py bug is fixed
-        delp = delp + flux_component(
-            fx, fy, rarea
-        )  # TODO: Put [0, 0, 0] on left when gt4py bug is fixed
-        pt[0, 0, 0] = pt / delp
+@gtscript.function
+def horizontal_relative_vorticity_from_winds_f(u, v, ut, vt, dx, dy, rarea, vorticity):
```

Comment:
Another option is to just have the stencil of the same name call this function rather than having duplicated code. But, like mentioned, this could be pushed into the VorticityVolumeMean test, or that tiny test can be removed.

Reply:
With the VorticityVolumeMean test removed, I got rid of the `_f` in the name of this function.